src/services/binance/service.ts

```diff
@@ -71,7 +71,7 @@ export const subscribeTransfers = (address: string) => {
       (e) => (e as WS.TransferEvent).data !== undefined
     )
     .pipe(
-      tap((msg) => console.log('transfer msg', msg)),
+      // tap((msg) => console.log('transfer msg', msg)),
```

Comment:
What purpose does this serve?

Reply:
It's for logging the value of this websocket message. By using [`tap`](https://rxjs.dev/api/operators/tap), you can do something with the value of the Observable and it will still return the same Obervable so you won't lost it for the next function (it's `map` in the next line). 

Here are some nice examples to explain `tap`: https://www.learnrxjs.io/learn-rxjs/operators/utility/do#examples